t/FHEM/14_LTECH/01_set.t

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use Test2::V0;
+use Test2::Tools::Compare qw{is};
+
+our %defs;
+our %attr;
+
+InternalTimer(time()+1, sub {
+	my $devicename = shift; 
+
+    subtest 'Protocol 31 - define LTECH_Test_11' => sub {
+		plan(1);
+        CommandDefine(undef,qq[$devicename LTECH 04444EFF]); 
```

Comment:
Das Gerät, kannst Du auch in fhem.cfg z.B. mit dem Namen LTECH_Test_11 anlegen und dann hier verwenden.
Als Argument wird es ja bereits übergeben.

```
my $name = shift;
my $hash = $defs{$devicename};

```

Reply:
Würde ich empfehlen.